tests/html5skin-test.js

```diff
@@ -346,6 +346,13 @@ describe('Controller', function() {
       expect(controller.state.screenToShow).toBe(CONSTANTS.SCREEN.LOADING_SCREEN);
     });
 
+    it('should reset isPlayingAd state when embed code is set after video has started', function() {
+      controller.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+      controller.state.initialPlayHasOccurred = true;
+      controller.onSetEmbedCode('newEmbedCode');
+      expect(controller.state.isPlayingAd).toBe(false);
```

Comment:
Looks good, I just think we should set to `controller.state.isPlayingAd` to `true` at the start of the test, otherwise we might not be testing that the value is being changed since it's `false` by default.

Reply:
oh, thank you!!

\+